starlark-rust/vscode/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846
+  // for the documentation about the extensions.json format
+  "recommendations": ["dbaeumer.vscode-eslint", "amodio.tsl-problem-matcher"]
```

Comment:
Is this a good thing to include? Do we really want to recommend it? Or is this normal for a vscode package?